Document the TMPDIR environment variable for better discovery (#308)
swiftlang/swiftly: Documentation/SwiftlyDocs.docc/swiftly-cli-reference.md

    $ swiftly install 5.7-snapshot
    $ swiftly install main-snapshot

Install whatever toolchain is currently selected, such as the the one in the .swift-version file:

    $ swiftly install

> NOTE: Toolchains are downloaded to a temporary file that is later cleaned up by the installation process. If these files are too big for your usual temporary directory you can choose another location by setting the `TMPDIR` environment variable.

Reviewer: ```suggestion
NOTE: Swiftly downloads toolchains to a temporary file that it later cleans during its installation process. If these files are too big for your system temporary directory, set another location by setting the `TMPDIR` environment variable.
```
